Whatsapp_Chat_Exporter/__main__.py

```diff
@@ -262,9 +262,9 @@ def main():
     )
     parser.add_argument(
         "--dont-filter-empty",
-        dest="filter_empty",
-        default=False,
-        action='store_true',
+        dest="disable_filter_empty",
```

Comment:
This is whole thing is still counter intuitive for me.
on the help it's written "By default, the exporter will not render"
But `default=True`
Means the filter empty is disabled by default and the empty chats we be rendered

Reply:
The ultimate goal is to create an intuitive method to determine if empty chats should be filtered in the code. For example, with the latest update (8069882):

```
if args.filter_empty:
  # This will filter out empty chats
else:
  # This will not filter out empty chats
```

Setting the default value to `True` means that `args.filter_empty` will trigger the filtering of empty chats. If the user sets the flag `--disable-filter-empty`, filtering will be disabled, thus the variable is `False`.